src/site/content/en/blog/get-installed-related-apps/index.md

```diff
@@ -141,13 +152,68 @@ your domain. Be sure to include the escaping characters.
 </string>
 ```
 
-### Test for the presence of your native app {: #test-native }
+Finally, publish your updated Android app to the Play store.
+
+### Define the relationship to an installed PWA {: #relationship-web }
+
+{% Aside %}
+**Coming Soon!**
+Expected in Chrome 81, websites can check if their PWA, served from the same
+domain is installed.
```

Comment:
It's not quite any WebAPK on the domain. You can only check for WebAPKs that control the page (the page includes a link to the manifest). Domains can have multiple WebAPKs that control different pages (different scopes).

Reply:
To make sure I understand this properly:

In Chrome 81:
* www.example.com/pwa/ (has manifest) **can** test www.example.com/pwa/
* www.example.com/marketing/ (no manifest) **cannot** test for www.example.com/pwa/

In Chrome 82:
* www.example.com/pwa/ (has manifest) **can** test www.example.com/pwa/
* www.example.com/marketing/ (no manifest) **can** test for www.example.com/pwa/ **if** the `assetlinks.json` file is present.

Is that correct?

Reply:
Yes, except for one minor detail, which is that to check for the related apps you must always have a manifest, since the list of related apps come from the manifest.

The reason for this is that example.com/maps PWA shouldn't have implicit permission for example.com/photos PWA, which is conceptually a different thing.

Reply:
Thanks - I've updated the messaging to be more clear about scope.